zerocopy-derive/tests/ui-nightly/enum.stderr

```diff
@@ -413,41 +413,53 @@ help: add `#![feature(trivial_bounds)]` to the crate attributes to enable
 9   + #![feature(trivial_bounds)]
     |
 
-error[E0277]: the trait bound `HasPadding<IntoBytes1, true>: ShouldBe<false>` is not satisfied
+error[E0277]: `IntoBytes1` has inter-field padding
    --> tests/ui-nightly/enum.rs:533:10
     |
 533 | #[derive(IntoBytes)]
-    |          ^^^^^^^^^ the trait `ShouldBe<false>` is not implemented for `HasPadding<IntoBytes1, true>`
+    |          ^^^^^^^^^ types with padding cannot implement `IntoBytes`
```

Comment:
This is ridiculously good. I wonder if we could even say how many bytes of inter-field padding there with const generics, but that's just yak shaving.

Reply:
Filed https://github.com/google/zerocopy/issues/1717 to track